env/.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,118 @@
+name: Deployments tests (E2E)
```

Comment:
We shouldn't need the .github directory and these files within since the ones needed should be moved to the .github at the base of the chainlink-testing-framework repo.

Reply:
argh you! merge issue...